### DISCUSSION TITLE
Can we add the state from SOLIX C800 Plus (A1754) to the hacs-anker solix Integration

### DISCUSSION THREAD

**Discussion by aiakos-k** [thomluther/ha-anker-solix#76] [Q&A] [answered]
Dear Thomluther and other,
thank you for this nice work on this great integration.

i use the SOLIX C800 Plus (A1754) and after install this integration and Log in to the Anker Acc.
i have in the Home Assistant:
1 Device with 4 Entity, see picture
![Entity](https://github.com/thomluther/hacs-anker-solix/assets/112417384/a23fc958-33bc-429e-b72b-181970107aca)

Can you help me / tell me or integrate the other Entity?
How?
I am willing to learn ;)

best regards,
Michael

**Comment by thomluther** [comment, marked as answer]
Hello @aiakos-k 
Standalone devices does not report much to the cloud and you won't get more data when you cannot add them to a power sysetm.
see #31

**Reply by aiakos-k** [reply]
Dear @thomluther,
this is not a good answer :D thank you for your very good work. Then i have to wait and saving for a large system :D
